JunitHomework/src/main/java/string/utility/NumbersModifierNotOk.java

```diff
@@ -0,0 +1,34 @@
+package string.utility;
```

Comment:
There s no need to have three classes and each class with one method. These methods are utility methods.
I think it would be enough to create a Utility class and have separate methods in that class.